contracts/src/Teleporter/tests/CheckAllowedRelayerTests.t.sol

```diff
@@ -6,27 +6,16 @@
 pragma solidity 0.8.18;
 
 import "./TeleporterMessengerTest.t.sol";
+import "forge-std/Test.sol";
 
-contract CheckIsAllowedRelayerTest is TeleporterMessengerTest {
-    // The state of the contract gets reset before each
-    // test is run, with the `setUp()` function being called
-    // each time after deployment.
-    function setUp() public virtual override {
-        TeleporterMessengerTest.setUp();
-    }
-
+contract CheckIsAllowedRelayerTest is TeleporterMessenger, Test {
```

Comment:
do we want to test `internal` functions of `TeleproterMessenger` by making derived contracts? I would lean towards `_checkIsAllowedRealyer` being tested in `TeleporterMessengerTests`

Reply:
What's the motivation for not wanting it to be in its own test suite? It makes it easier to find and isolate IMO.

I don't love making a derived contract to test `internal` functions, but I think it's the easy option that provides the needed coverage.

Reply:
Good with the derived contract and own test suite, just want to make sure consistent with how other internal functions were tested.

Reply:
This was actually the first internal method in the `TeleporterMessenger` contract. 👍 

They theoretically could be tested through various inputs to the public/external methods that call them, but in this case I think I'd rather keep the isolated test coverage for it specifically.